rust/examples/src/bin/place_order.rs

```diff
@@ -0,0 +1,175 @@
+use std::str::FromStr;
+
+use ellipsis_client::EllipsisClient;
+use phoenix::state::Side;
+use phoenix_sdk::sdk_client::SDKClient;
+use solana_client::nonblocking::rpc_client::RpcClient;
+use solana_sdk::account::Account;
+use solana_sdk::commitment_config::CommitmentConfig;
+#[allow(unused_imports)]
+use solana_sdk::{
+    instruction::Instruction,
+    program_pack::Pack,
+    pubkey::Pubkey,
+    signature::{read_keypair_file, Keypair},
+    signer::Signer,
+};
+use spl_associated_token_account::instruction::create_associated_token_account;
+use spl_token::state::Mint;
+#[tokio::main]
+async fn main() {
+    // Connect to the Solana network and identify the market you want to interact with, using phoenix-cli, for example.
+    let network_url = "https://api.devnet.solana.com";
+    let market_pubkey = Pubkey::from_str("CS2H8nbAVVEUHWPF5extCSymqheQdkd4d7thik6eet9N").unwrap();
+
+    let trader = Keypair::new();
+    let rpc_client =
+        RpcClient::new_with_commitment(network_url.to_string(), CommitmentConfig::confirmed());
+    let client = EllipsisClient::from_rpc(rpc_client, &trader).unwrap();
+    println!("Trader pubkey: {}", trader.pubkey());
+    // Create an SDKClient instance using the recommended method. If you only needed a specific known market, you can also use SDKClient::new_with_market_keys
+    let sdk = SDKClient::new_from_ellipsis_client_with_all_markets(client)
+        .await
+        .unwrap();
+    // Alternatively, read from keypair file for the payer
+    // let path = "~/.config/solana/id.json";
+    // let auth_payer = read_keypair_file(&*shellexpand::tilde(path)).unwrap();
+
+    println!("trader {}", sdk.trader);
+
+    // On devnet, as in this example, we airdrop the base and quote token for the market to the trader.
```

Comment:
"These instructions only work on devnet. They trigger an airdrop from the devnet-token-faucet (link) for the base and quote tokens for the supplied market, for MM testing."